chain/pkg/obi/schema_test.go

```diff
@@ -0,0 +1,83 @@
+package obi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type EmptySchema struct{}
+
+type Color struct {
+	code string `code` //missing obi
+}
```

Comment:
Can we also add test for `bytes`?

Reply:
Added test case for bytes

Reply:
This struct is expected to be mock missing OBI tag struct.